### DISCUSSION TITLE
v9: Combobox allow input value with spaces

### DISCUSSION THREAD

**Discussion by TomSoftwerft** [microsoft/fluentui#26445] [Q&A] [answered]
Hey, I need to implement an address autocompletion where the user can type something like _My Street 123_. At the moment the combobox selects the focused value if the user hits the spacebar. Is there a way I can achieve the desired behaviour with the current implementation of the combobox? If it is not currently possible, are there plans to support this?

**Comment by micahgodbolt** [comment, marked as answer]
The default behavior right now is that space selects the focused item in the listbox. This is pretty consistent across other inputs, so we did not want to deviate there.

**Comment by TomSoftwerft** [comment]
solved by https://github.com/microsoft/fluentui/pull/27025
